docs/tutorial/force_calibration.rst

```diff
@@ -53,7 +53,9 @@ This model can be found in :class:`~.PassiveCalibrationModel`. It is calibrated
 
     P(f) = \frac{D}{\pi ^ 2 \left(f^2 + f_c ^ 2\right)} g(f, f_{diode}, \alpha)
 
-where :math:`D` corresponds to the diffusion constant, :math:`f` the frequency and :math:`f_c` the fitted cutoff. The second term :math:`g` takes into account the slower response of the position detection system and is given by:
+where :math:`D` corresponds to the diffusion constant, :math:`f` the frequency and :math:`f_c` the fitted cutoff.
```

Comment:
I am more familiar with the term  'corner frequency' than with 'fitted cutoff'. Maybe good to say cutoff/corner frequency or something, such that people with different backgrounds understand the meaning of this term.

Reply:
Yes. Good point! :)
Amended!